### DISCUSSION TITLE
[next] error - Failed to load next.config.js

### DISCUSSION THREAD

**Discussion by sickdaflip** [graphcommerce-org/graphcommerce#1725] [Q&A] [answered]
commerce@inet:/var/customers/webs/commerce/my-project$ yarn dev
yarn run v1.22.11
$ concurrently -k -n codegen,next 'graphql-codegen -w' 'next dev'
[next] ready - started server on 0.0.0.0:3000, url: http://localhost:3000
[next] info  - Loaded env from /var/customers/webs/commerce/my-project/.env
[next] error - Failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
[next] Error: next-transpile-modules - an unexpected error happened when trying to resolve "@graphcommerce/magento-graphcms". Are you sure the name of the module you are trying to transpile is correct, and it has a package.json with a "main" or an "exports" field?
[next] Error: Can't resolve '@graphcommerce/magento-graphcms/package.json' in '/var/customers/webs/commerce/my-project'
[next]     at getPackageRootDirectory (/var/customers/webs/commerce/my-project/node_modules/next-transpile-modules/src/next-transpile-modules.js:123:15)
[next]     at Array.map (<anonymous>)
[next]     at withTM (/var/customers/webs/commerce/my-project/node_modules/next-transpile-modules/src/next-transpile-modules.js:132:34)
[next]     at /var/customers/webs/commerce/my-project/node_modules/@graphcommerce/next-config/dist/withGraphCommerce.js:53:83
[next]     at Object.<anonymous> (/var/customers/webs/commerce/my-project/next.config.js:66:26)
[next]     at Module._compile (node:internal/modules/cjs/loader:1159:14)
[next]     at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
[next]     at Module.load (node:internal/modules/cjs/loader:1037:32)
[next]     at Module._load (node:internal/modules/cjs/loader:878:12)
[next]     at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
[next] next dev exited with code 1
--> Sending SIGTERM to other processes..
[codegen] graphql-codegen -w exited with code SIGTERM
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

what am I doing wrong?

**Comment by paales** [comment, marked as answer]
@sickdaflip We _just_ released a new version, can you try that? https://github.com/graphcommerce-org/graphcommerce/releases/tag/release-1666879622347

**Comment by sickdaflip** [comment]
@paales Thanks for your quick reply!

`git clone https://github.com/graphcommerce-org/graphcommerce.git --branch release-1666879622347 --single-branch`

works wonderfully

**Reply by paales** [reply]
Nice! It has also been released to main, so the regular instructions should be working as well!
